bin/picca_export.py

```diff
@@ -240,6 +242,48 @@ def main(cmdargs):
         'R-parallel', 'R-transverse', 'Redshift', 'Correlation',
         'Covariance matrix', 'Distortion matrix', 'Number of pairs'
     ]
+
+    # Check if we need blinding and apply it
+    if 'BLIND' in data_name or blinding != 'none':
+        if blinding != 'corr_yshift':
+            print('Only strategy E called "corr_yshift" is allowed for now.'
+                  ' This will be applied automatilly.')
```

Comment:
As I say above I think this is ok, but we should consider resetting at this point the variable "blinding":
blinding = 'corr_yshift'